src/config/Blocks.jsx

```diff
@@ -79,6 +79,14 @@ defineMessages({
     id: 'html',
     defaultMessage: 'HTML',
   },
+  leadimage: {
+    id: 'leadimage',
+    defaultMessage: 'Lead Image Field',
+  },
+  listing: {
+    id: 'listing',
+    defaultMessage: 'Elenco',
```

Comment:
@nzambello ups, I think this slipped in, right?

Reply:
@sneridagh whoopsie sorry it was a mistake.